src/AppBundle/Resources/views/Ndr/Action/_answers.html.twig

```diff
@@ -4,138 +4,154 @@
-                    <span class="error">
-                        {{ 'form.actionGiveGiftsToClient.label' | trans(transOptions) }}
-                    </span>
-                </td>
-                <td>
-                    <span class="error-message">{{ 'pleaseAnswer' | trans({}, 'common' ) }}</span>
-                </td>
-            {% else %}
-                <td>
+
+{# step 1 #}
+<dl class="govuk-summary-list columns-2 push--bottom">
+    <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+            {{ 'question' | trans({}, 'common') }}
+        </dt>
+        <dd class="govuk-summary-list__key">
+            {{ 'answer' | trans({}, 'common') }}
+        </dd>
+        <dd class="govuk-summary-list__actions"></dd>
+    </div>
+
+    <div class="govuk-summary-list__row behat-region-give-gifts-client">
+        {% if ndr.actionGiveGiftsToClient is null %}
+            <dt class="govuk-summary-list__key">
+                <span class="error">
                     {{ 'form.actionGiveGiftsToClient.label' | trans(transOptions) }}
-                </td>
-                <td>
-                    {{ ndr.actionGiveGiftsToClient | capitalize }}
-                </td>
-            {% endif %}
-            <td class="change-answer">
-                <a href="{{ path('ndr_actions_step', {'ndrId': ndr.id, 'step': 1, 'from': 'summary'}) }}" class="behat-link-edit action-link">{{ 'edit' | trans({}, 'common' ) }}</a>
-            </td>
-        </tr>
-        {% if ndr.actionGiveGiftsToClient == 'yes' %}
-            <tr class="behat-region-give-gifts-client-details">
-                <td>{{ 'form.actionGiveGiftsToClientDetails.label' | trans(transOptions) }}</td>
-                <td>{{ ndr.actionGiveGiftsToClientDetails | nl2br }}</td>
-                <td class="change-answer">
-                    <a href="{{ path('ndr_actions_step', {'ndrId': ndr.id, 'step': 1, 'from': 'summary'}) }}" class="action-link">{{ 'edit' | trans({}, 'common' ) }}</a>
-                </td>
-            </tr>
+                </span>
+            </dt>
+            <dd class="govuk-summary-list__value">
+                <span class="error-message">{{ 'pleaseAnswer' | trans({}, 'common' ) }}</span>
+            </dd>
+        {% else %}
+            <dt class="govuk-summary-list__key">
+                {{ 'form.actionGiveGiftsToClient.label' | trans(transOptions) }}
+            </dt>
+            <dd class="govuk-summary-list__value">
+                {{ ndr.actionGiveGiftsToClient | capitalize }}
+            </dd>
         {% endif %}
-    </tbody>
-
-</table>
+        <dd class="govuk-summary-list__actions">
+            <a href="{{ path('ndr_actions_step', {'ndrId': ndr.id, 'step': 1, 'from': 'summary'}) }}" class="behat-link-edit govuk-link">
+                {{ 'edit' | trans({}, 'common' ) }}
+            </a>
+        </dd>
+    </div>
+    <div class="govuk-summary-list__row behat-region-give-gifts-client-details">
```

Comment:
Looks like this should be in a `{% if ndr.actionGiveGiftsToClient == 'yes' %}` statement